test/TestCase.cpp

```diff
@@ -168,3 +168,16 @@ EVMVersionRestrictedTestCase::EVMVersionRestrictedTestCase(std::string const& _f
 	processEVMVersionSetting();
 	processBytecodeFormatSetting();
 }
+
+std::ostream& solidity::frontend::test::operator<<(std::ostream& _output, CompileViaYul _compileViaYul)
+{
+	switch (_compileViaYul)
+	{
+	case CompileViaYul::False: _output << "false"; break;
+	case CompileViaYul::True: _output << "true"; break;
+	case CompileViaYul::Also: _output << "also"; break;
+	case CompileViaYul::onlyOnEOF: _output << "onlyOnEOF"; break;
```

Comment:
The enum value should be capitalized to match the others:

```suggestion
	case CompileViaYul::OnlyOnEOF: _output << "onlyOnEOF"; break;
```